src/components/worker/WorkerMasterPanel.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { withTheme, withStyles } from "@material-ui/core/styles";
+import { Paper, Grid, Typography, Divider, Checkbox, FormControlLabel } from "@material-ui/core";
+import {
+  formatMessage,
+  FormattedMessage,
+  PublishedComponent,
+  FormPanel,
+  TextInput,
+  withModulesManager,
+} from "@openimis/fe-core";
+
+const styles = (theme) => ({
+  paper: theme.paper.paper,
+  tableTitle: theme.table.title,
+  item: theme.paper.item,
+  fullHeight: {
+    height: "100%",
+  },
+});
+
+class WorkerMasterPanel extends FormPanel {
+  render() {
+    const {
+      intl,
+      classes,
+      edited,
+      title = "Insuree.title",
+      titleParams = { label: "" },
+      readOnly = true,
+      edited_id
+    } = this.props;
+    return (
+      <Grid container>
+        <Grid item xs={12}>
+          <Paper className={classes.paper}>
+            <Grid container className={classes.tableTitle}>
+              <Grid item xs={3} container alignItems="center" className={classes.item}>
+                <Typography variant="h5">
+                  <FormattedMessage module="insuree" id={title} values={titleParams} />
+                </Typography>
+              </Grid>
+            </Grid>
+            <Divider />
+            <Grid container className={classes.item}>
+              <Grid item xs={4} className={classes.item}>
+                <PublishedComponent
+                  pubRef="insuree.InsureeNumberInput"
+                  module="insuree"
+                  label="Insuree.nationalId"
+                  required={true}
+                  readOnly={readOnly}
+                  value={edited?.chfId}
+                  edited_id={edited_id}
+                  onChange={(v) => this.updateAttribute("chfId", v)}
+                />
+              </Grid>
+              <Grid item xs={4} className={classes.item}>
+                <TextInput
+                  module="insuree"
+                  label="Insuree.lastName"
+                  required={true}
+                  readOnly={readOnly}
+                  value={!!edited && !!edited.lastName ? edited.lastName : ""}
```

Comment:
This could be replaced by:
`
value={edited?.lastName ?? " "}
`

Reply:
done